experimental/lib/MyProgramProof.fst

```diff
@@ -9,7 +9,7 @@ open Armada.Statement
 open Armada.Transition
 open Armada.Type
 open FStar.List.Tot.Base
-open FStar.Tactics
+open FStar.Tactics.V2
```

Comment:
@mtzguido From what I can tell, you've been migrating a lot of code in F* to V2.  Is this what we should do here as well?

Reply:
Hi Gabriel. Yes, these changes look perfect to me